vnext/src/Libraries/Components/AccessibilityInfo/AccessibilityInfo.windows.js

```diff
@@ -151,7 +151,9 @@ const AccessibilityInfo = {
    * See https://reactnative.dev/docs/accessibilityinfo.html#setaccessibilityfocus
    */
   setAccessibilityFocus: function(reactTag: number): void {
-    legacySendAccessibilityEvent(reactTag, 'focus');
+    if (NativeAccessibilityInfo) {
```

Comment:
haven't looked at the base file, but is there a change to remove this fork since we're adding the implementation?